egs/callhome_diarization/v1/run.sh

```diff
@@ -0,0 +1,175 @@
+#!/bin/bash
+# Copyright 2017  David Snyder
+#           2017  Matthew Maciejewski
+# Apache 2.0.
+#
+# This is still a work in progress, but implements something similar to
+# Greg Sell's and Daniel Garcia-Romero's iVector-based diarization system
+# in https://www.dropbox.com/s/bj5bc6brtzt52u4/slt_gks_dgr.pdf?dl=0 .
+# The main difference is that we haven't implemented the VB resegmentation
+# yet.
+
+. cmd.sh
+. path.sh
+set -e
+mfccdir=`pwd`/mfcc
+vaddir=`pwd`/mfcc
+num_components=2048
+ivector_dim=128
+
+# Prepare a collection of NIST SRE data. This will be used to train the UBM,
+# iVector extractor and PLDA model.
+local/make_sre.sh data
+
+# Prepare SWB for UBM and iVector extractor training.
+local/make_swbd2_phase2.pl /export/corpora5/LDC/LDC99S79 \
+                           data/swbd2_phase2_train
+local/make_swbd2_phase3.pl /export/corpora5/LDC/LDC2002S06 \
+                           data/swbd2_phase3_train
+local/make_swbd_cellular1.pl /export/corpora5/LDC/LDC2001S13 \
+                             data/swbd_cellular1_train
+local/make_swbd_cellular2.pl /export/corpora5/LDC/LDC2004S07 \
+                             data/swbd_cellular2_train
+
+# NOTE: For now, this is not a propery data prep script.  It assumes that the
+# data was already prepared elsewhere (e.g., in
+# /home/dsnyder/a16/a16/dsnyder/SCALE17/callhome), and copies the files to
+# the local directories here.
+local/make_callhome.sh /home/dsnyder/a16/a16/dsnyder/SCALE17/callhome data/
```

Comment:
@david-ryan-snyder The callhome data preparation script is just copying from your directory on the CLSP servers. Can this be changed to a script that operates on the LDC release of the corpus? Is there anything special in how you created that directory?

Reply:
No, this is just a placeholder for a real dataprep script. I doubt you'll need to do anything fancy.

Problem is that I'm not sure what the correct LDC dataset is for this. You might need to do some searching.

Reply:
I believe the LDC ID is LDC2001S97 (https://catalog.ldc.upenn.edu/LDC2001S97).

The portion used for diarization can be found on the CLSP grid here, I believe:
/export/corpora/NIST/LDC2001S97/sid00sg1/data/